ledfx/api/websocket.py

```diff
@@ -297,6 +300,36 @@ def audio_stream_data_handler(self, message):
             message.get("data").values(), dtype=np.float32
         )
 
+    @websocket_handler("audio_stream_data_v2")
+    def audio_stream_data_base64_handler(self, message):
+        # Max value for signed 16-bit values.
+        MAX_VAL = 32767
+        if not ACTIVE_AUDIO_STREAM:
+            return
+
+        client = message.get("client")
```

Comment:
Not familiar enough with websockets - does this need protection at all?

Reply:
This was already there, I did not add this, so I did not touch it.

Reply:
No worries - just asking since if we both have eyes on/around it, it's generally worth sniffing around see if there's anything else that could be "fixed".

Reply:
As I understand, this is to make sure that there is an actual active webaudio stream. Because I think the websocket can (theoretically) receive audio data without an actual webaudio stream being open, which you don't want, so this protects from that scenario.